packages/host/tests/acceptance/code-submode/inspector-test.ts

```diff
@@ -803,6 +838,120 @@ module('Acceptance | code submode | inspector tests', function (hooks) {
     }
   });
 
+  test<TestContextWithSave>('can duplicate an instance in same realm', async function (assert) {
+    assert.expect(10);
+    let operatorModeStateParam = stringify({
+      stacks: [[]],
+      submode: 'code',
+      codePath: `${testRealmURL}Pet/vangogh.json`,
+    })!;
+    await visit(
+      `/?operatorModeEnabled=true&operatorModeState=${encodeURIComponent(
+        operatorModeStateParam,
+      )}`,
+    );
+    await waitForCodeEditor();
+    await waitFor(`[data-test-action-button="Duplicate"]`);
+    await click('[data-test-action-button="Duplicate"]');
+    await waitFor(
+      `[data-test-create-file-modal][data-test-ready] [data-test-realm-name="Test Workspace B"]`,
+    );
+    await percySnapshot(assert);
```

Comment:
This seems superfluous to me since the modal with realm chooser is already covered by other snapshots:

<img width="748" alt="Boxel 2023-12-15 11-27-15" src="https://github.com/cardstack/boxel/assets/43280/ea310798-45e9-4d1a-a4b9-1f461ad6a0a7">